classes/class-admin.php

```diff
@@ -469,14 +484,10 @@ public function admin_enqueue_scripts( $hook ) {
 				'wp_stream_live_updates',
 				array(
 					'current_screen'      => $hook,
-					'current_page'        => isset( $_GET['paged'] ) ? esc_js( $_GET['paged'] ) : '1', // WPCS: CSRF ok.
-					// input var okay, CSRF okay
-					'current_order'       => isset( $_GET['order'] ) ? esc_js( $_GET['order'] ) : 'desc', // WPCS: CSRF ok.
-					// input var okay, CSRF okay
-					'current_query'       => wp_stream_json_encode( $_GET ), // WPCS: CSRF ok.
-					// input var okay, CSRF okay
-					'current_query_count' => count( $_GET ), // WPCS: CSRF ok.
-					// input var okay, CSRF okay
+					'current_page'        => isset( $_GET['paged'] ) ? esc_js( $_GET['paged'] ) : '1', // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+					'current_order'       => isset( $_GET['order'] ) ? esc_js( $_GET['order'] ) : 'desc', // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Comment:
Could we check that it's one of the allowed values of `order` before setting it? Do we support anything apart from `desc` and `asc`?

Reply:
I agree, if there are only `asc` and `desc` we can do a check for one and manually set this instead of excepting any value.